### PR TITLE
Fix for REGISTRY-3141

### PR DESCRIPTION
**Pull request by mushthaq33**

### CODE REVIEW

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 1]
```diff
@@ -64,17 +64,17 @@
 import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
 import org.wso2.carbon.utils.multitenancy.MultitenantUtils;
 
+import java.io.StringReader;
+import java.util.*;
```
Comment (prasa7) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Remove the * import.

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java [hunk 2]
```diff
@@ -1794,7 +1794,15 @@ public static void setDefaultLifeCycle(String path, String aspect, Registry regi
                 "updater", "updater!", "tags", "content", "mediaType", "mediaType!", "lcName", "lcState");
 
         String[] tempList = criteria.split("&");
-        for(String temp : tempList) {
+        List<String> finalTempList = new ArrayList<>();
+        for (int i = 0; i < tempList.length; i++) {
+            if (tempList[i].contains("=")) {
+                finalTempList.add(tempList[i]);
+            } else {
+                finalTempList.set(i - 1, finalTempList.get(i - 1) + "&" + tempList[i]);
```
Comment (prasa7) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: What would be the case if i = 1 ?

Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Fixed in the following PR - https://github.com/wso2/carbon-governance/pull/156

File: components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java (hunk 1 above)
Comment (mushthaq33) [inline components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java]: Fixed with the following PR - https://github.com/wso2/carbon-governance/pull/254

**Pull request merged**